<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ViewTexture</class>
 <widget class="QWidget" name="ViewTexture">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>631</width>
    <height>463</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Просмотр доступных текстур</string>
  </property>
  <property name="styleSheet">
   <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(46, 47, 48);</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>3</number>
   </property>
   <property name="topMargin">
    <number>3</number>
   </property>
   <property name="rightMargin">
    <number>3</number>
   </property>
   <property name="bottomMargin">
    <number>3</number>
   </property>
   <property name="spacing">
    <number>3</number>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="TextureViewLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>420</width>
       <height>420</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="FiltersLabel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>Consolas</family>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Фильтры: </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>4</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="ResolutionSortLabel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>Consolas</family>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="text">
        <string>По разрешению: </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="SearchsizeQComboBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>85</width>
         <height>0</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Consolas</family>
         <pointsize>9</pointsize>
        </font>
       </property>
       <item>
        <property name="text">
         <string>None</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>512X512</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>1024X1024</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2048X2048</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>4096X4096</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>4</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="SearchNameLabel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>Consolas</family>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="text">
        <string>По имени: </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="SearchNameQLineEdit">
       <property name="font">
        <font>
         <family>Consolas</family>
         <pointsize>9</pointsize>
        </font>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>1</number>
     </property>
     <property name="sizeConstraint">
      <enum>QLayout::SetFixedSize</enum>
     </property>
     <item>
      <widget class="QListWidget" name="TexturesQListWidget">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>0</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Consolas</family>
        </font>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>300</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Consolas</family>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="title">
        <string>Свойства</string>
       </property>
       <widget class="QLabel" name="TypeLabel">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>27</y>
          <width>31</width>
          <height>31</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>Consolas</family>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Тип: </string>
        </property>
       </widget>
       <widget class="QLabel" name="BumpMappingLabel">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>200</y>
          <width>181</width>
          <height>18</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Consolas</family>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Bump mapping:</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="pathTobumpmapQLineEdit">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>220</y>
          <width>181</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Consolas</family>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QComboBox" name="ChangeSizeMatFileQComboBox">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>90</y>
          <width>61</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Consolas</family>
          <pointsize>9</pointsize>
         </font>
        </property>
        <item>
         <property name="text">
          <string>KBs</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>MBs</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>GBs</string>
         </property>
        </item>
       </widget>
       <widget class="QLineEdit" name="SizeMaterialQLineEdit">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>60</y>
          <width>121</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Consolas</family>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="text">
         <string>%FE_1024x1024%</string>
        </property>
        <property name="frame">
         <bool>true</bool>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QLineEdit" name="pathToNormalMapQLineEdit">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>170</y>
          <width>181</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Consolas</family>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="acceptDrops">
         <bool>true</bool>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="frame">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="Line" name="line_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>120</y>
          <width>181</width>
          <height>16</height>
         </rect>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
       <widget class="QLineEdit" name="SizeMaterialFileQLineEdit">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>90</y>
          <width>61</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Consolas</family>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="text">
         <string>%FILE_SIZE%</string>
        </property>
        <property name="frame">
         <bool>true</bool>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QLabel" name="SizeTextureLabel">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>60</y>
          <width>61</width>
          <height>31</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>Consolas</family>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Размер: </string>
        </property>
       </widget>
       <widget class="QLineEdit" name="TypeQLineEdit">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>30</y>
          <width>121</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Consolas</family>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="frame">
         <bool>true</bool>
        </property>
        <property name="readOnly">
         <bool>false</bool>
        </property>
       </widget>
       <widget class="QLabel" name="NormalMappingLabel">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>150</y>
          <width>181</width>
          <height>18</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Consolas</family>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Normal mapping:</string>
        </property>
       </widget>
       <widget class="QLabel" name="PathToAdditionalFilesLabel">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>130</y>
          <width>181</width>
          <height>18</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Consolas</family>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Пути к доп. файлам:</string>
        </property>
       </widget>
       <widget class="QLabel" name="FileLabel">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>90</y>
          <width>61</width>
          <height>31</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>Consolas</family>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="text">
         <string>файла:</string>
        </property>
       </widget>
       <widget class="QPushButton" name="OkPushButton">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>260</y>
          <width>71</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Consolas</family>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="text">
         <string>ОК</string>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="OKPushButton">
       <property name="font">
        <font>
         <family>Consolas</family>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="text">
        <string>ОК</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="CancelPushButton">
       <property name="font">
        <font>
         <family>Consolas</family>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Отмена</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
